src/scss/layout/_generalLayout.scss

```diff
@@ -1,10 +1,66 @@
 .general-layout {
   display: flex;
   flex-direction: row;
+  background-color: rgb(241, 242, 241);
 }
 
-@media all and (max-width: 720px) {
+.general-title{
+  margin-top: 90px;
```

Comment:
It will be better if you separate general setting and customized setting for specific width range.
It may be like this:
```scss
.general-title {
  width: auto;
  height: auto;
  text-align: left;
  color: rgb(112, 112, 112);
}

// 1280 < size
@media all and (min-width: 1280px) {
  .general-title {
    margin-top: 90px;
    margin-left: 60px;
    font-size: 40px;
  }
}
```

Reply:
done